stackslib/src/clarity_vm/clarity.rs

```diff
@@ -1360,7 +1360,6 @@ impl<'a, 'b> ClarityBlockConnection<'a, 'b> {
                     Value::UInt(u128::from(first_block_height)),
                     Value::UInt(u128::from(pox_prepare_length)),
                     Value::UInt(u128::from(pox_reward_cycle_length)),
-                    Value::UInt(u128::from(pox_rejection_fraction)),
```

Comment:
Please also remove the declaration of `pox_rejection_fraction`.

Reply:
done